WindowsServerDocs/administration/windows-commands/gpt.md

```diff
@@ -22,22 +22,29 @@ On basic GUID partition table (gpt) disks, assigns the gpt attribute(s) to the p
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| attributes=<n> | Specifies the value for the attribute that you want to apply to the partition with focus. The gpt attribute field is a 64-bit field that contains two subfields. The higher field is interpreted only in the context of the partition ID, while the lower field is common to all partition IDs. Accepted values include:<br /><br />-   **0x0000000000000001**. Specifies that the partition is required by the computer to function properly.<br />-   **0x8000000000000000**. Specifies that the partition will not receive a drive letter by default when the disk is moved to another computer or when the disk is seen for the first time by a computer.<br />-   **0x4000000000000000**. Hides a partition's volume. That is, the partition will not be detected by the mount Manager.<br />-   **0x2000000000000000**. Specifies that the partition is a shadow copy of another partition.<br />-   **0x1000000000000000**. Specifies that the partition is read-only. This attribute prevents the volume from being written to.<br /><b />for more information about these attributes, see the attributes section at [create_PARTITION_PARAMETERS Structure](https://go.microsoft.com/fwlink/?LinkId=203812) (<https://go.microsoft.com/fwlink/?LinkId=203812>). |
```

Comment:
Cosmetic / codestyle: 1) unwarranted bold formatting of the last line, 2) uncapitalized first word of the sentence.
```suggestion
| attributes=<n> | Specifies the value for the attribute that you want to apply to the partition with focus. The gpt attribute field is a 64-bit field that contains two subfields. The higher field is interpreted only in the context of the partition ID, while the lower field is common to all partition IDs. Accepted values include:<br /><br />-   **0x0000000000000001**. Specifies that the partition is required by the computer to function properly.<br />-   **0x8000000000000000**. Specifies that the partition will not receive a drive letter by default when the disk is moved to another computer or when the disk is seen for the first time by a computer.<br />-   **0x4000000000000000**. Hides a partition's volume. That is, the partition will not be detected by the mount Manager.<br />-   **0x2000000000000000**. Specifies that the partition is a shadow copy of another partition.<br />-   **0x1000000000000000**. Specifies that the partition is read-only. This attribute prevents the volume from being written to.<br /><br />For more information about these attributes, see the attributes section at [create_PARTITION_PARAMETERS Structure](https://go.microsoft.com/fwlink/?LinkId=203812) (<https://go.microsoft.com/fwlink/?LinkId=203812>). |
```
BTW, is it common to refer to other pages twice, by showing the URL as well as marking part of the text as a link?

Reply:
> BTW, is it common to refer to other pages twice, by showing the URL as well as marking part of the text as a link?

Oh, this is not the case. It has been fixed in https://github.com/MicrosoftDocs/windowsserverdocs/pull/3671/commits/eebd535acf8c77ee85a2c1a6fd2a2e4a3ca42623 . Thank you.